PyInstaller/utils/win32/versioninfo.py

```diff
@@ -581,6 +581,20 @@ def SetVersion(exenm, versionfile):
         txt = fp.read()
         fp.close()
         vs = eval(txt)
+
+    pe = pefile.PE(exenm, fast_load=True)
+    overlay_before = pe.get_overlay()
+    pe.close()
+
     hdst = win32api.BeginUpdateResource(exenm, 0)
     win32api.UpdateResource(hdst, pefile.RESOURCE_TYPE['RT_VERSION'], 1, vs.toRaw())
     win32api.EndUpdateResource (hdst, 0)
+
+    pe = pefile.PE(exenm, fast_load=True)
```

Comment:
1. I assumes this only needs to be done `if overlay_before`.
1. Please add a short comment like "check if the overlay is still present".